pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -593,24 +593,29 @@ class Build {
         }
 
         def appOptions="customJtx=${excludeRoot}/jenkins/jck_run/jdk${jdkVersion}/${excludePlat}/temurin.jtx"
-
-        if (configureArguments.contains('--enable-headless-only=yes')) {
+        def targets
+        if (configureArguments.contains('--enable-headless-only=yes') || "${platform}" == 'ppc64_aix' || "${platform}" == 'sparcv9_solaris' || "${platform}" == 'x86-64_solaris') {
             // Headless platforms have no auto-manuals, so do not exclude any tests
             appOptions=""
```

Comment:
ah AIX and Solaris, should not black appOptions

Reply:
Updated